package.json

```diff
@@ -55,17 +55,22 @@
 		"autogypi.json"
 	],
 	"devDependencies": {
+		"@types/node": "^8.10.17",
 		"ava": "^0.25.0",
 		"clang-format": "^1.2.2",
 		"humanize": "0.0.9",
 		"husky": "^0.14.3",
-		"proc-stats": "0.0.4"
+		"prettier": "^1.12.1",
+		"proc-stats": "0.0.4",
+		"typescript": "^2.8.3"
```

Comment:
Why we need typescript as a devdep? It will allow us to check types in example, tests etc?
If so, could you add an entry in scripts to do this check?